core/test/Test/Pos/Core/ExampleHelpers.hs

```diff
@@ -636,31 +636,56 @@ exampleGenesisAvvmBalances =
                      ,Coin {getCoin = 37343863242999412})
                      ]) }
   where
-    fstRedKey = "\254\156\235\217{]\130W\183LfJ\240"
+    fstRedKey =
+        "\39906\174753\164215\154426\53802\53634\49873\133644\34608\
+        \\144009\37388\53695\147588\134892\161995\46300\22042\149930\
+        \\13038\38802\71172\29846\11842\136013\66358\63976\64104\
+        \\4099\48285\22172\74421\44313"
     sndRedKey =
-        "\254\156\235\217{]\130W\183LfJ\240\RS\224"
+        "\22940\29914\144828\177644\52019\16682\5563\142278\22269\41464\
+        \\64899\153034\134491\51443\9384\49583\41181\16586\45161\55231\
+        \\127978\3521\6732\67841\152627\1796\53978\28040\68524\45059\
+        \\46383\4326"
```

Comment:
We have hex encoding\decoding helpers somewhere. Is i worth using them for this? IMO hex is much more readable and uniform than this stuff.

Reply:
Hex is more readable than what you have :)

Reply:
I just force-pushed a new version which uses `Serokell.Util.Base16` to encode the ByteStrings. Let me know if that's more like what you imagined.